community/io/src/main/java/org/neo4j/io/fs/DefaultFileSystemAbstraction.java

```diff
@@ -133,9 +134,9 @@ public void deleteRecursively( File directory ) throws IOException
     }
 
     @Override
-    public boolean renameFile( File from, File to ) throws IOException
+    public void renameFile( File from, File to, CopyOption... copyOptions ) throws IOException
     {
-        return FileUtils.renameFile( from, to );
+        FileUtils.renameFile( from, to, copyOptions );
```

Comment:
Because `path( from )` is what delegates to provided file system in delegate.
I guess we could let `DefaultFileSystemAbstraction` do
```
    @Override
    public void renameFile( File from, File to, CopyOption... copyOptions ) throws IOException
    {
        Files.move( from.toPath(), to.toPath(), copyOptions );
    }
```